### PR TITLE
Update constants path for PS Versions >= 8.1

### PR DESCRIPTION
**Pull request by FabienPapet**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Documentation for https://github.com/PrestaShop/PrestaShop/pull/29750/

### CODE REVIEW

**Review by matks — COMMENTED** [review]
All good, now waiting for core PR to be merged

File: content/maintainers-guide/releasing-prestashop/preliminary-tasks.md [hunk 1]
```diff
@@ -48,6 +48,10 @@ Check the following files and update them if necessary:
     const MINOR_VERSION = 6;
     const RELEASE_VERSION = 2;
     ```
+    
+{{% notice tip %}}
+Starting of Prestashop 8.1, version contants have now changed location and are now located in `src/Core/Version.php`
+
```
Comment (eternoendless) [inline content/maintainers-guide/releasing-prestashop/preliminary-tasks.md:54]:
```suggestion
{{% notice tip %}}
Starting with Prestashop 8.1, version constants are located in `src/Core/Version.php`.
{{% /notice %}}

```

**Review by eternoendless — CHANGES_REQUESTED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by FabienPapet** [comment]
Branch has been merged, can we merge this ?

**Comment by eternoendless** [comment]
Thank you @FabienPapet

**Pull request merged**